<ui version="4.0" >
 <class>StatisticHistogramFormClass</class>
 <widget class="QWidget" name="StatisticHistogramFormClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>StatisticHistogramForm</string>
  </property>
  <widget class="QTabWidget" name="tabWidgetInspect" >
   <property name="geometry" >
    <rect>
     <x>240</x>
     <y>410</y>
     <width>321</width>
     <height>121</height>
    </rect>
   </property>
   <property name="currentIndex" >
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_5" >
    <attribute name="title" >
     <string>しきい値</string>
    </attribute>
    <widget class="QSpinBox" name="EditOKDotL" >
     <property name="geometry" >
      <rect>
       <x>140</x>
       <y>50</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum" >
      <number>99999999</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="EditOKDotH" >
     <property name="geometry" >
      <rect>
       <x>230</x>
       <y>50</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum" >
      <number>99999999</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_17" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape" >
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text" >
      <string>OK輝度範囲</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_18" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>50</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape" >
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text" >
      <string>OKドット数</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_19" >
     <property name="geometry" >
      <rect>
       <x>140</x>
       <y>10</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape" >
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text" >
      <string>暗側</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_20" >
     <property name="geometry" >
      <rect>
       <x>230</x>
       <y>10</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape" >
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text" >
      <string>明側</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="EditOKLengthL" >
     <property name="geometry" >
      <rect>
       <x>140</x>
       <y>70</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum" >
      <number>99999999</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_32" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>70</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape" >
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text" >
      <string>これ以上の長さでＮＧ</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="EditOKLengthH" >
     <property name="geometry" >
      <rect>
       <x>230</x>
       <y>70</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum" >
      <number>99999999</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="EditBrightWidthL" >
     <property name="geometry" >
      <rect>
       <x>140</x>
       <y>30</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="EditBrightWidthH" >
     <property name="geometry" >
      <rect>
       <x>230</x>
       <y>30</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab" >
    <attribute name="title" >
     <string>膨張・収縮</string>
    </attribute>
    <widget class="QLabel" name="label_28" >
     <property name="geometry" >
      <rect>
       <x>30</x>
       <y>70</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape" >
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text" >
      <string>収縮ドット数</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkEliminateAcnode" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="geometry" >
      <rect>
       <x>30</x>
       <y>30</y>
       <width>121</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text" >
      <string>孤立点の除去</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="EditErosionDotCount" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="geometry" >
      <rect>
       <x>250</x>
       <y>70</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum" >
      <number>255</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_27" >
     <property name="geometry" >
      <rect>
       <x>30</x>
       <y>50</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape" >
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text" >
      <string>膨張ドット数</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkUseMorphology" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>膨張・収縮をする</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="EditDilationDotCount" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="geometry" >
      <rect>
       <x>250</x>
       <y>50</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum" >
      <number>255</number>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxMorphologyApplyTo" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="geometry" >
      <rect>
       <x>170</x>
       <y>10</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text" >
       <string>暗側・明側両方</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>暗側のみ</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>明側のみ</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_7" >
    <attribute name="title" >
     <string>属性</string>
    </attribute>
    <widget class="QCheckBox" name="checkBModeEnabled" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text" >
      <string>検査有効</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBModeAbsoluteBright" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>30</y>
       <width>141</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text" >
      <string>絶対輝度</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBModeNGCluster" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>50</y>
       <width>141</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text" >
      <string>塊検査</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBModeWhiteMask" >
     <property name="geometry" >
      <rect>
       <x>160</x>
       <y>30</y>
       <width>141</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text" >
      <string>白マスク</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBModeBlackMask" >
     <property name="geometry" >
      <rect>
       <x>160</x>
       <y>50</y>
       <width>141</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text" >
      <string>黒マスク</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_8" >
    <attribute name="title" >
     <string>補正</string>
    </attribute>
    <widget class="QLabel" name="label_26" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>70</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape" >
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text" >
      <string>明側　最大補正輝度量</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_25" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>40</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape" >
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text" >
      <string>暗側　最大補正輝度量</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="EditAdjustWhite" >
     <property name="geometry" >
      <rect>
       <x>220</x>
       <y>70</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum" >
      <number>255</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="EditAdjustBlack" >
     <property name="geometry" >
      <rect>
       <x>220</x>
       <y>40</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum" >
      <number>255</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_5" >
   <property name="geometry" >
    <rect>
     <x>80</x>
     <y>450</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text" >
    <string>ライブラリ名称</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>450</y>
     <width>62</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text" >
    <string>ライブラリＩＤ</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>500</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QRadioButton" name="radioButtonDarkSide1" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>4</y>
      <width>51</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Dark</string>
    </property>
    <property name="checked" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonLightSide1" >
    <property name="geometry" >
     <rect>
      <x>100</x>
      <y>4</y>
      <width>51</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Light</string>
    </property>
    <property name="checked" >
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="EditLibID" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>470</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>12345</string>
   </property>
   <property name="readOnly" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>180</x>
     <y>10</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>マスター画像</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame" >
   <property name="geometry" >
    <rect>
     <x>570</x>
     <y>410</y>
     <width>181</width>
     <height>161</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLabel" name="label_31" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>0</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text" >
     <string>選択ブロックの二値化後情報</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_29" >
    <property name="geometry" >
     <rect>
      <x>120</x>
      <y>30</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text" >
     <string>明側</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>60</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text" >
     <string>ドット数</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_30" >
    <property name="geometry" >
     <rect>
      <x>70</x>
      <y>30</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text" >
     <string>暗側</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelResultLight" >
    <property name="geometry" >
     <rect>
      <x>120</x>
      <y>60</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text" >
     <string>123</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelResultDark" >
    <property name="geometry" >
     <rect>
      <x>70</x>
      <y>60</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text" >
     <string>123</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonCalc" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>130</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>計算</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_7" >
   <property name="geometry" >
    <rect>
     <x>350</x>
     <y>10</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>検査画像</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8" >
   <property name="geometry" >
    <rect>
     <x>510</x>
     <y>10</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>しきい値</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonChangeLib" >
   <property name="geometry" >
    <rect>
     <x>180</x>
     <y>470</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>...</string>
   </property>
  </widget>
  <widget class="QFrame" name="frameLineGraph" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>40</y>
     <width>731</width>
     <height>321</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Plain</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frameLayer" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>410</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Plain</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditLibName" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>80</x>
     <y>470</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>XXX</string>
   </property>
   <property name="readOnly" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonLoadFromLibrary" >
   <property name="geometry" >
    <rect>
     <x>200</x>
     <y>580</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>ライブラリから読み出し</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonSaveToLibrary" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>580</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>ライブラリに保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonReflectAllBlocks" >
   <property name="geometry" >
    <rect>
     <x>200</x>
     <y>540</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>同一種類のブロックに反映</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidgetParts" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>540</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="currentIndex" >
    <number>2</number>
   </property>
   <widget class="QWidget" name="page" >
    <widget class="QPushButton" name="ButtonRelrectOnlyBlock" >
     <property name="geometry" >
      <rect>
       <x>0</x>
       <y>0</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text" >
      <string>このブロックだけに反映</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3" >
    <widget class="QPushButton" name="ButtonRelrectSaveParts" >
     <property name="geometry" >
      <rect>
       <x>0</x>
       <y>0</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text" >
      <string>部品保存</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2" />
  </widget>
  <widget class="QPushButton" name="ButtonClose" >
   <property name="geometry" >
    <rect>
     <x>380</x>
     <y>540</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>閉じる</string>
   </property>
  </widget>
  <widget class="QFrame" name="frameTarget" >
   <property name="geometry" >
    <rect>
     <x>320</x>
     <y>10</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Plain</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frameThreshold" >
   <property name="geometry" >
    <rect>
     <x>480</x>
     <y>10</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Plain</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frameMaster" >
   <property name="geometry" >
    <rect>
     <x>150</x>
     <y>10</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Plain</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditAvr" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>380</x>
     <y>390</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditSigma" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>470</x>
     <y>390</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_21" >
   <property name="geometry" >
    <rect>
     <x>380</x>
     <y>370</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text" >
    <string>平均値</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_22" >
   <property name="geometry" >
    <rect>
     <x>470</x>
     <y>370</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text" >
    <string>標準偏差</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
